roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/clusterrole-ingress-nginx.yml.j2

```diff
@@ -3,7 +3,9 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   name: ingress-nginx
-  namespace: {{ ingress_nginx_namespace }}
```

Comment:
good that you deleted this. clusterroles are cluster-wide, not namespaced

Reply:
yes, reference upstream changes and bugfix it ;-)